update-data.py

```diff
@@ -105,9 +77,35 @@ def insert_people(cursor, people):
                        ]
                       )
         for stint in person['stints']:
-            logging.debug("  Stint: {}, batch {}, {} - {}".format(
+            batch_id = None
+            batch = stint.get('batch')
+
+            if (batch):
+                batch_id = batch.get('id')
+
+                if (batch_id not in processed_batches and
+                    not person.get('is_faculty') == 'true' and
```

Comment:
I believe the `is_faculty` field is part of the [deprecated people-by-batch API](https://github.com/recursecenter/wiki/wiki/Recurse-Center-API#list-the-people-in-a-batch-deprecated-in-favor-of-profiles-search), and is not present on the new [profiles API](https://github.com/recursecenter/wiki/wiki/Recurse-Center-API#profiles) which we're using now. Plus, I think we're already filtering them out by the `if (batch):` line, as they don't have that field in their stints.

Reply:
Yes - I was still looking at the old People API instead of Profiles. After our pairing session, this is resolved, and the check for faculty isn't needed.